calculator.js

```diff
@@ -0,0 +1,31 @@
+//Add
+function [
+      function addition(num1, num2){
+            return num1 + num2;
+            console.log(addition(num1, num2));
+      }
+      function subtraction (num1, num2){
+            return num1 - num2;
+            console.log(subtraction(num1, num2));
+      }
+      function multiply (num1, num2){
+            return num1 * num2;
+            console.log(multiply(num1, num2));
```

Comment:
Since when the webpage loads the script errors out. I'm assuming you tested the parts of this script individually by copying & pasting them into the chorme devtools console. Did you see the console.log? Why not?